README.md

```diff
@@ -48,31 +48,31 @@ The benefits of this library are:
     -   See [`./perf`](./perf/) folder for a micro-benchmark
         -   Only 4 times slower than a native multi-threaded transformer
         -   Fastest compared to non-native (JavaScript or Wasm)
-    -   No new JavaScript code is generated, instead it re-uses slices of the existing source string
-    -   This is particularly true if your program is already generating the TypeScript `SourceFile` object because it can [be reused](#bring-your-own-ast), and producing the AST is the most time consuming part.
+    -   No new JavaScript code is generated; instead, it re-uses slices of the existing source string
+    -   This is particularly true if your program is already generating the TypeScript `SourceFile` object, because it can [be reused](#bring-your-own-ast) -- and producing the AST is the most time consuming part.
 -   100% JavaScript runtime
     -   No [Wasm](https://webassembly.org)
     -   No [native-addons](https://nodejs.org/api/addons.html)
     -   No [child process](https://nodejs.org/api/child_process.html)
 -   It is small
-    -   Less than 700 lines of code and one dependency ([`typescript`](https://www.npmjs.com/package/typescript))
-    -   By doing so little the code should be relatively easy to maintain
+    -   Less than 700 lines of code and one dependency ([`TypeScript`](https://www.npmjs.com/package/typescript))
+    -   By doing so little, the code should be relatively easy to maintain
 -   Delegates the parsing to the [official TypeScript parser](https://github.com/microsoft/TypeScript/wiki/Using-the-Compiler-API)
--   No need for additional SourceMap processing. See ["where are my SourceMaps?"](#where-are-my-sourcemaps)
+-   No need for additional SourceMap processing; see ["where are my SourceMaps?"](#where-are-my-sourcemaps)
 
-:information_source: Not all TypeScript syntax is supported (see [unsupported syntax](#unsupported)). There is also no down leveling, the JavaScript is preserved as is.
+:information_source: Not all TypeScript syntax is supported (see [unsupported syntax](#unsupported)). There is also no down-leveling; the JavaScript is preserved as is.
 
 ## Contents
 
 -   [Installing](#installing)
 -   [API](#api)
 -   [Node.js Loader](#nodejs-loader)
--   [Source Maps](#where-are-my-sourcemaps)
+-   [SourceMaps](#where-are-my-sourcemaps)
 -   [Implementation details](#does-it-really-just-blank-out-all-the-type-annotations)
--   [Unsupported syntax](#unsupported)
+-   [Unsupported Syntax](#unsupported)
```

Comment:
```suggestion
-   [Unsupported Syntax](#unsupported-syntax)
```

Fixes up the link as the headers have changed (would be nice if CI could validate the anchors)

Reply:
Oh that "nice catch" comment I added was misguided, this is the catch!